src/python/dxpy/nextflow/nextflow_builder.py

```diff
@@ -66,7 +66,7 @@ def parse_extra_args(extra_args):
     if not brief:
         print("Created Nextflow pipeline %s" % (applet_id))
     else:
-        print(applet_id)
+        print(dict(id=applet_id))
```

Comment:
Is there a difference between this and [this](https://github.com/dnanexus/dx-toolkit/blob/master/src/python/dxpy/scripts/dx_build_app.py#L1042) (which prints [this](https://github.com/dnanexus/dx-toolkit/blob/master/src/python/dxpy/scripts/dx_build_app.py#L758))? I.e. using dict vs dumping json? 

I made a note about tests in the ticket, please check it. I think the app needs to be updated and its update synced with the platform release.

Reply:
I think it should be pretty much the same, because it outputs a string. I will change it to json dumps, just to make it same as we already had it in codebase.

Yes, I will create PR for that

Reply:
There is actually no need to change anything in the NPI, as it uses --json anyway (that prints way more information about the applet in json format.)

Reply:
What about the test?